GameOfLife/Controller/GameManager.cs

```diff
@@ -1,29 +1,70 @@
-﻿namespace GameOfLife
+﻿using Newtonsoft.Json;
+
+namespace GameOfLife
 {
     /// <summary>
     /// Management of the game.
     /// </summary>
-    public class GameManager
+    public class GameManager : IGameManager
     {
         /// <summary>
         /// Starts the game.
         /// </summary>
         public void Play()
         {
-            UserInterface userInterface = new UserInterface();
-            GameView gameView = new GameView();
             UserInterface.WelcomeScreen();
-            
-            int column = userInterface.GetValidUserInput(UserInterface.inputColomnMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
-            int row = userInterface.GetValidUserInput(UserInterface.inputRowMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
-            Game game = new Game(column, row);
-            gameView.View(game.gameField);
+
+            int column = UserInterface.GetValidUserInput(GameMessages.inputColomnMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
+            int row = UserInterface.GetValidUserInput(GameMessages.inputRowMessage, GameParameters.minInputValue, GameParameters.maxInputValue);
+            Game game = new(column, row);
+            GameView.View(game.GameField);
+
             while (true)
             {
                 game.GetNextGeneration();
-                gameView.View(game.gameField);
+                GameView.View(game.GameField);
+                Console.WriteLine($"Iteration: {game.CountOfIteration}");
+                Console.WriteLine($"Live cells: {game.CountOfLiveCells}");
+                Console.WriteLine(GameMessages.pauseGame);
```

Comment:
It would be better not to use Console object outside UserInterface class.

It looks that displaying messages should be part of GameView class, because it is part of displaying data on screen.

Reply:
26.-28. line moved to DisplayGame.